lib/dao.js

```diff
@@ -3130,8 +3130,9 @@ function(data, options, cb) {
   for (var i = 0, n = idNames.length; i < n; i++) {
     var idName = idNames[i];
     if (data[idName] !== undefined && !idEquals(data[idName], inst[idName])) {
-      var err = new Error(g.f('{{id}} property (%s) ' +
-        'cannot be updated from %s to %s'), idName, inst[idName], data[idName]);
+      var err = new Error(g.f('{{id}} cannot be updated from ' +
+        '%s to %s when forceId is set to true',
```

Comment:
@kjdelisle @strongloop/sq-lb-apex  `forceId` should not be translated. it should be `{{forceId}}`